jaeger_client/constants.py

```diff
@@ -65,3 +65,6 @@
 # of traces per second.
 # noinspection SpellCheckingInspection
 SAMPLER_TYPE_LOWER_BOUND = 'lowerbound'
+
+# max length for tag values. Longer values will be truncated.
+MAX_TAG_VALUE_LENGTH = 1024
```

Comment:
Why 1024? 
What is the purpose of curtailing the length? 
Is it better to have `MAX_TAG_VALUE_BYTES` instead? (Because unicode chars can span multiple bytes?)

Reply:
python client does not have logic to calculate the size of the UDP packet, instead it just limits the number of spans per packet. Having unbounded tag/log values increases the risk of exceeding the packet size. We've seen huge SQL statements in production.

Reply:
If we want to figure out the size in bytes, I think it is more useful to have `MAX_TAG_VALUE_BYTES` instead of length, because the length would be shorter than the number of bytes when using some unicode characters.

Reply:
why does it matter? It's an approximation anyway.

Reply:
I see, I didn't realize that it was an approximation.